docs/usage/8.md

```diff
@@ -0,0 +1,298 @@
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: cam-controller
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - ${local_registry}/${local_registry_ns}/openshift-migration-controller-rhel8
+    source: ${source_registry}/${source_registry_ns}/openshift-migration-controller-rhel8
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: cam-ui
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - ${local_registry}/${local_registry_ns}/openshift-migration-ui-rhel8
+    source: ${source_registry}/${source_registry_ns}/openshift-migration-ui-rhel8
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: cam-velero
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - ${local_registry}/${local_registry_ns}/openshift-migration-velero-rhel8
+    source: ${source_registry}/${source_registry_ns}/openshift-migration-velero-rhel8
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: cam-plugin
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - ${local_registry}/${local_registry_ns}/openshift-migration-plugin-rhel8
+    source: ${source_registry}/${source_registry_ns}/openshift-migration-plugin-rhel8
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: cam-helper
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - ${local_registry}/${local_registry_ns}/openshift-migration-velero-restic-restore-helper-rhel8
+    source: ${source_registry}/${source_registry_ns}/openshift-migration-velero-restic-restore-helper-rhel8
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: roperator
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - ${local_registry}/${local_registry_ns}/rook-ceph-rhel8-operator
+    source: ${source_registry}/${source_registry_ns}/rook-ceph-rhel8-operator
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: moperator
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - ${local_registry}/${local_registry_ns}/mcg-rhel8-operator
+    source: ${source_registry}/${source_registry_ns}/mcg-rhel8-operator
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: ooperator
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - ${local_registry}/${local_registry_ns}/ocs-rhel8-operator
+    source: ${source_registry}/${source_registry_ns}/ocs-rhel8-operator
+---
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: noobaa
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - ${local_registry}/${local_registry_ns}/mcg-core-rhel8
+    source: ${source_registry}/${source_registry_ns}/mcg-core-rhel8
+
+EOF
+
+oc create -f image-content-source-policy.yml
+```
+
+## Step 9c: Mirror the Images
```

Comment:
Mirror the images -> Mirror the Images required by the CAM, OCS, Rook, and MCG operators